src/plans/D3MTrueFiV1Plan.t.sol

```diff
@@ -112,15 +111,13 @@ contract D3MTrueFiV1PlanTest is AddressRegistry, D3MPlanBaseTest {
 
     function _setUpTrueFiDaiPortfolio() internal {
         portfolioFactory = PortfolioFactoryLike(MANAGED_PORTFOLIO_FACTORY_PROXY);
-        lenderVerifier = WhitelistVerifierLike(GLOBAL_WHITELIST_LENDER_VERIFIER);
 
         // Grant address(this) auth access to factory and lender verifier
         hevm.store(MANAGED_PORTFOLIO_FACTORY_PROXY, bytes32(uint256(0)), bytes32(uint256(uint160(address(this)))));
         portfolioFactory.setIsWhitelisted(address(this), true);
-        hevm.store(GLOBAL_WHITELIST_LENDER_VERIFIER, bytes32(uint256(0)), bytes32(uint256(uint160(address(this)))));
```

Comment:
we need to add ```// set address of d3mTestPool to true in whitelist mapping in global whitelist lender verifier
        hevm.store(GLOBAL_WHITELIST_LENDER_VERIFIER, keccak256(abi.encode(d3mTestPool, 2)), bytes32(uint256(1)));``` here otherwise plan tests will fail